package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "nodemon --exec babel-node src/index.js",
-    "data": "nodemon --exec babel-node src/database/test_data/test_data1.js"
+    "data": "nodemon --exec babel-node src/database/test_data/test_data3.js"
```

Comment:
why we need nodemon here?

Reply:
test_data3 will not work without data2 and 1 
you should handle it in the right way

Reply:
babel wont run without nodemon